examples/AutomatedSummaryReport/Ballerina.toml

```diff
@@ -0,0 +1,5 @@
+[package]
+org = "wso2"
```

Comment:
Not sure if the org name is ideal.

Reply:
I would have thought something like ballerina-samples would be better. Or even connector-samples.